site/content/en/docs/concepts/cluster_queue.md

```diff
@@ -349,8 +349,8 @@ To limit the amount of resources that a ClusterQueue can lend in the cohort,
 you can set the `.spec.resourcesGroup[*].flavors[*].resource[*].lendingLimit`
 [quantity](https://kubernetes.io/docs/reference/kubernetes-api/common-definitions/quantity/) field.
 
+{{< feature-state state="stable" for_version="v0.6.0+" >}}
```

Comment:
```suggestion
{{< feature-state state="alpha" for_version="v0.6" >}}
```

This matches how Kubernetes documents features https://kubernetes.io/docs/reference/access-authn-authz/validating-admission-policy/

For the current state of features, you can check https://kueue.sigs.k8s.io/docs/installation/#change-the-feature-gates-configuration or https://github.com/kubernetes-sigs/kueue/blob/aeb13cf6b1733f578b2e301b2d506e13e4776285/pkg/features/kube_features.go#L105